Gemfile

```diff
@@ -9,6 +9,7 @@ gem 'json', '~> 2.1.0'
 gem 'plek', '2.1.1'
 
 gem 'govuk_frontend_toolkit', '~> 7.2.0'
+gem 'govuk_publishing_components', github: 'alphagov/govuk_publishing_components'
```

Comment:
I'm assuming that the version bump of this is inbound so we can use the published gem?